adapter/config/default_config.go

```diff
@@ -177,18 +177,23 @@ var defaultConfig = &Config{
 			},
 		},
 		JwtGenerator: jwtGenerator{
-			Enabled:               false,
-			Encoding:              "base64",
-			ClaimDialect:          "http://wso2.org/claims",
-			ConvertDialect:        false,
-			Header:                "X-JWT-Assertion",
-			SigningAlgorithm:      "SHA256withRSA",
-			EnableUserClaims:      false,
-			GatewayGeneratorImpl:  "org.wso2.carbon.apimgt.common.gateway.jwtgenerator.APIMgtGatewayJWTGeneratorImpl",
-			ClaimsExtractorImpl:   "org.wso2.carbon.apimgt.impl.token.ExtendedDefaultClaimsRetriever",
-			PublicCertificatePath: "/home/wso2/security/truststore/mg.pem",
-			PrivateKeyPath:        "/home/wso2/security/keystore/mg.key",
-			TokenTTL:              3600,
+			Enabled:              false,
+			Encoding:             "base64",
+			ClaimDialect:         "http://wso2.org/claims",
+			ConvertDialect:       false,
+			Header:               "X-JWT-Assertion",
+			SigningAlgorithm:     "SHA256withRSA",
+			EnableUserClaims:     false,
+			GatewayGeneratorImpl: "org.wso2.carbon.apimgt.common.gateway.jwtgenerator.APIMgtGatewayJWTGeneratorImpl",
+			ClaimsExtractorImpl:  "org.wso2.carbon.apimgt.impl.token.ExtendedDefaultClaimsRetriever",
+			TokenTTL:             3600,
+			Keypair: []KeyPair{
+				{
+					PublicCertificatePath: "/home/wso2/security/truststore/mg.pem",
+					PrivateKeyPath:        "/home/wso2/security/keystore/mg.key",
+					UseForSigning:         true,
+				},
```

Comment:
We have to support the backward compatibility in this case. Basically existing configuration should work with this new implementation until we change the configurations.

Reply:
ack